lib/crypto/crypto_keys_subr.c

```diff
@@ -180,23 +183,23 @@ crypto_keys_subr_import_RSA_priv(RSA ** key, const uint8_t * buf, size_t buflen)
 	if (import_BN(&n, &buf, &buflen))
 		goto err1;
```

Comment:
No memory allocated, and no memory will be freed (I assume you've checked that).  The only reason to have `err1` and `err2` is if you're going to continue with `err3`, `err4`, et cetera... the whole point of setting them to NULL is so that we can always jump to the same place.